File: mobile/iOS/GuestJini/GuestJini/Global/Model/Responses/KBRatingPercentResponse.swift
//
//  KBRatingPercentResponse.swift
//  GuestJini
//
//  Created by Prasanna Kumar Pete on 22/01/20.
//  Copyright © 2020 Prasanna Kumar Pete. All rights reserved.
//

import Foundation
class KBRatingPercentResponse: Decodable {
    public var likedPercent:Double? = 0
    public var disLikedPercent:Double?  = 0
    public var likedCount:Int = 0
    public var disLikedCount:Int = 0
    public var success:Bool = false
    public var error:String = ""
}

File: mobile/android/GuestJini/app/src/main/java/com/carmel/guestjini/Networking/KnowledgeBase/KBRatingPercentResponse.java
package com.carmel.guestjini.Networking.KnowledgeBase;

public class KBRatingPercentResponse {
    public Double likedPercent;
    public Double disLikedPercent;
    public Integer likedCount;
    public Integer disLikedCount;
    private Boolean success;
    private String error;

    public Double getLikedPercent() {
        return likedPercent;
    }

    public void setLikedPercent(Double likedPercent) {
        this.likedPercent = likedPercent;
    }

    public Double getDisLikedPercent() {
        return disLikedPercent;
    }

    public void setDisLikedPercent(Double disLikedPercent) {
        this.disLikedPercent = disLikedPercent;
    }

    public Integer getLikedCount() {
        return likedCount;
    }

    public void setLikedCount(Integer likedCount) {
        this.likedCount = likedCount;
    }

    public Integer getDisLikedCount() {
        return disLikedCount;
    }

    public void setDisLikedCount(Integer disLikedCount) {
        this.disLikedCount = disLikedCount;
    }

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public String getError() {
        return error;
    }

    public void setError(String error) {
        this.error = error;
    }
}
